docker-compose-remote.yml

```diff
@@ -60,12 +60,12 @@ services:
           memory: 200M
 
   pg:
+    profiles: ["gotenzymes"]
     build:
       context: pg
+      dockerfile: prod.Dockerfile
     restart: always
     command: postgres -c stats_temp_directory=/tmp
-    volumes:
```

Comment:
do not use volumes anymore, data is kept in the image and will be updated on a rebuild of this